language-support/hs/bindings/test/DA/Ledger/Sandbox.hs

```diff
@@ -0,0 +1,123 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Sandbox ( -- Run a sandbox for testing on a dynamically selected port
```

Comment:
Why not `DA.Ledger.Sandbox` as the module name? Having the module name and path not mix is often a poor idea.

Reply:
Oversight. Fixed.